libbeat/common/transport/tlscommon/ca_pinning.go

```diff
@@ -51,7 +51,7 @@ func verifyCAPin(hashes []string, verifiedChains [][]*x509.Certificate) error {
 
 // Fingerprint takes a certificate and create a hash of the DER encoded public key.
 func Fingerprint(certificate *x509.Certificate) string {
-	hash := sha256.Sum256(certificate.RawSubjectPublicKeyInfo)
+	hash := sha256.Sum256(certificate.Raw)
```

Comment:
@simitt, I've tried the change you suggested here and my tests have rejected the certificate

Reply:
Could you be a bit more specific what rejected the certificate means? And did you convert the ES fingerprint from `hex` to `base64` before configuring it?

Reply:
And since it is self-sigend, you need to also configure the CA with the current libbeat code. I mentioned this in my first comment, as it might be a problem if only the fingerprint is passed down from Fleat.

Reply:
both hex and basic should still be accepted right? Can we know if certs are hex or basic in order to avoid convertion when not needed?